src/Controller/BlogController.php

```diff
@@ -38,12 +38,24 @@ public function blog(FigureRepository $repoFigure)
       
         $figureSlides = $repoFigure->findBy(array(), array('id' => 'DESC'), 3);
 
-        $figures = $repoFigure->findBy(array(), array('id' => 'DESC'), null, 3);
+        $newfigures = $repoFigure->findBy(array(), array('id' => 'DESC'), 6, 3);
         
         return $this->render('blog/posts.html.twig', [
             'current_member' => $current_member,
-            'figures' => $figures,
-            'figureSlides' => $figureSlides
+            'figureSlides' => $figureSlides,
+            'newfigures' => $newfigures
+        ]);
+    }
+
+    /**
+     * @Route("/{starting}", name="more_figures", requirements={"starting": "\d+"})
+     */
+    public function moreFigures(FigureRepository $repoFigure, $starting = 3)
```

Comment:
```suggestion
    public function moreFigures(FigureRepository $repoFigure, $starting = 3): Response
```
Indiquer le paramètre de retour des fonctions en PHP